cmd/releasego/add-retry-instructions.go

```diff
@@ -131,3 +100,69 @@ func handleAddRetryInstructions(p subcmd.ParseFunc) error {
 
 	return nil
 }
+
+// Match env vars like "poll1buildId=123" with the number and value in groups. If there are
+// multiple "=", treat the leftmost one as the separator. Ignore case.
+var reg = regexp.MustCompile(`(?i)poll(\d+)(.+?)=(.*)`)
+
+type envArg struct {
+	Value string
+	Name  string
+	Index int
+}
+
+func newEnvArg(envVar string) (*envArg, error) {
+	matches := reg.FindStringSubmatch(envVar)
+	if len(matches) > 0 {
+		fmt.Printf("Found polling env var %q. Matches: %#v\n", envVar, matches)
+		num, err := strconv.Atoi(matches[1])
+		if err != nil {
+			return nil, fmt.Errorf("match 1 is not an int: %w", err)
```

Comment:
Consider putting `matches[1]` in the error string so we know what didn't parse.

Reply:
The value actually shows up in `err` already, e.g. `strconv.Atoi: parsing "not-a-number": invalid syntax`, `strconv.Atoi: parsing "79832459834523534534532425246246246246593847": value out of range`